crates/bevy_core_pipeline/src/overlay/mod.rs

```diff
@@ -0,0 +1,150 @@
+use bevy_app::{App, Plugin};
+use bevy_asset::{load_internal_asset, HandleUntyped};
+use bevy_diagnostic::{Diagnostics, FrameTimeDiagnosticsPlugin};
+use bevy_ecs::{
+    entity::Entity,
+    prelude::With,
+    system::{Commands, Local, Query, Res},
+};
+use bevy_input::{keyboard::KeyCode, Input};
+use bevy_reflect::TypeUuid;
+use bevy_render::{
+    extract_component::ExtractComponentPlugin,
+    prelude::Shader,
+    render_graph::{RenderGraph, SlotInfo, SlotType},
+    renderer::RenderQueue,
+    RenderApp, RenderStage,
+};
+
+mod camera_overlay;
+mod overlay_node;
+mod pipeline;
+use bevy_time::Time;
+use bevy_utils::{Duration, Instant};
+pub use camera_overlay::{CameraOverlay, CameraOverlayBundle};
+
+use crate::overlay::{overlay_node::graph, pipeline::OverlayPipeline};
+
+use self::overlay_node::DiagnosticOverlayBuffer;
+
+pub const OVERLAY_SHADER_HANDLE: HandleUntyped =
```

Comment:
I feel like this would be better organized as an associated constant. Perhaps under the plugin?

Reply:
I copied the style of all other const handles, I would prefer to keep consistency